docs/4_2_co-simulation_api.adoc

```diff
@@ -469,18 +469,20 @@ Once all events are handled and `newDiscreteStatesNeeded == fmi3False`, the FMU
 In this case, a new step can be started from the current communication point time.
 
 <<fmi3GetClock>>::
-The status of <<clock,`clocks`>> can be inquired by this function.
+The status of <<outputClock,`output clocks`>> and <<localClock,`local clocks`>> can be inquired by this function during *Event Mode* and *Intermediate Update Mode*.
 
 <<fmi3SetClock>>::
 For <<inputClock,`input clocks`>>, <<fmi3SetClock>> is called after entering *Event Mode* to set the activation status of <<clock,`clocks`>>.
-For both <<input>> and trigerred <<clock,`clocks`>>, this function can be called several times, only if recomputations of clock state are needed during *Event Mode*.
+This function can be called several times, only if recomputations of clock state are needed during *Event Mode*.
 
 <<fmi3GetIntervalDecimal>> & <<fmi3GetIntervalFraction>>::
-For both <<outputClock,`output clocks`>> and <<inputClock,`input clocks`>> it is allowed to call these functions during *Event Mode*.
+For <<outputClock,`output clocks`>> and <<localClock,`local clocks`>> it is allowed to call these functions during *Event Mode* and *Intermediate Update Mode*.
+These functions must be called only at the first activation of <<perodic>> <<outputClock,`output clocks`>>s.
+For <<perodic,`aperiodic`>> <<outputClock,`output clocks`>>, these functions must be called at every activation _[to inquire the next clock tick]_.
```

Comment:
> For <<perodic,`aperiodic`>> <<outputClock,`output clocks`>>, these functions must be called at every activation _[to inquire the next clock tick]_.

Should it really be "must be called" and not "can be called"? 
We should make it clearer that the clock ticks of input clocks that are triggered by the output clock are meant here:

> _[to inquire when triggered <<inputClock,`input clocks`>> must tick ]_

Reply:
Agreed, fixed as suggested - thank you!